src/OpenLoco/EditorController.cpp

```diff
@@ -0,0 +1,81 @@
+#include "EditorController.h"
+#include "Ui/WindowManager.h"
+
+using namespace OpenLoco::Interop;
+using namespace OpenLoco::Ui;
+using namespace OpenLoco::Ui::Windows;
+
+namespace OpenLoco::EditorController
+{
+    static loco_global<Step, 0x009C8714> _editorStep;
+
+    // 0x0043D7DC
+    void init()
+    {
+        call(0x0043D7DC);
+    }
+
+    Step getCurrentStep()
+    {
+        return (Step)*_editorStep;
```

Comment:
Why the cast? Can't you just return `_editorStep` that should handle all of the dereferencing.